examples/ziti-end-to-end/README.md

```diff
@@ -0,0 +1,98 @@
+```
+ziti-controller run --log-formatter pfxlog $ZITI_SOURCE/ziti/etc/ctrl.with.edge.yml
+```
+
+We then need to initialize the controller with an admin user. For ease of testing I just used `admin admin admin` for my short local test.
+
+```
+ziti agent controller init <username> <password> <name-of-user>
+```
+
+You can then login to the edge via `ziti edge login` and putting in your credentials.
+
+### Policies
+Next we create all of the relevant service policies:
+
+```
+ziti edge create service-policy dial-simple Dial --service-roles '#simple' --identity-roles '#simple-client'
+ziti edge create service-policy bind-simple Bind --service-roles '#simple' --identity-roles '#simple-server'
+```
+
+And the Edge router policies:
+
+```
+ziti edge create edge-router-policy simple-client --identity-roles '#simple-client' --edge-router-roles '#all'
+ziti edge create edge-router-policy simple-server --identity-roles '#simple-server' --edge-router-roles '#all'
+ziti edge create service-edge-router-policy simple --service-roles '#simple' --edge-router-roles '#all'
+```
+
+Now we create and enroll our client user
+
+```
+ziti edge create identity service simple-client -o simple-client.jwt -a simple-client,simple
+ziti edge enroll -j simple-client.jwt -o simple-client.json
+```
+
+And then create and enroll the server
+
+```
+ziti edge create identity service simple-server -o simple-server.jwt -a simple-server,simple
+ziti edge enroll -j simple-server.jwt -o simple-server.json
+```
+
+All that's left to do for this portion is to create the service
+```
+ziti edge create service echo -a simple
+```
+
+Now you can optionally delete the two jwt's we generated. We will not be using them in the rest of the example.
+### Edge Router
+Next we create and run the edge router:
+
+```
+ziti edge create edge-router edge-router -o edge-router.jwt -t
+```
+
+Just like with the controller I'm using an example `edge.router.yml` found in the source repository.
+```
+ziti-router enroll -j edge-router.jwt ${ZITI_SOURCE}/ziti/etc/edge.router.yml
+```
+
+Now we run the edge router like so! The controller host and port info is found in the controller config file you used.
+```
+CONTROLLER_HOST=localhost CONTROLLER_PORT=6262 ZITI_EDGE_PORT=3022 LINK_LISTENER_PORT=4022 exec $GOPATH/bin/ziti-router run ${DEBUG} --debug-ops -v --log-formatter pfxlog ${ZITI_SOURCE}/ziti/etc/edge.router.yml
+```
+
+## go-httpbin Server
+All we need to do to run the server is pass in the relevant flags. This assumes you ran all above commands in the same directory, which you should see `simple-server.json`. If not then point the `-ziti-identity` flag to that file wherever it is.
+```
+go-httpbin -ziti -ziti-identity ${PWD}/simple-server.json -ziti-name echo
+```
+
+## Client
+Running the client is all the same, just pass in the relevant files like above.
+```
+go-httpbin-client -header k=v -header k=v2 -query y=m -ziti -ziti-identity ${PWD}/simple-client.json -ziti-name echo post test
```

Comment:
clarity: I like that we're using the `/post` operation of go-httpbin to serve an HTTP echo. May I suggest examples here that express the function of headers and search query params? For example, `HEADER00=df9999677f0c0a97` and `query_param11=lorem-ipsum` and `-ziti-name my_http_post_echo_service post '{"test-data-key": "test-data-value"}'`.

Reply:
Just to clarify: Do you mean have an example hi-lighting each? Right now the current "example" shows all of them

Reply:
I meant to suggest replacing the generic placeholders like `y=m` with something more meaningful like `query_param11=lorem-ipsum`.

Reply:
Ohhhh I completely misunderstood. Can do!